tox.ini

```diff
@@ -24,6 +24,8 @@ deps =
     pytest54: pytest==5.4.*
     pytest60: pytest==6.0.*
     pytest61: pytest==6.1.*
+    pytest62: pytest==6.2.*
+    pytestmaster: git+https://github.com/pytest-dev/pytest.git@master#egg=pytest
```

Comment:
I like testing against `master`.
This change should also be applied to `.github/workflows/test.yml` so CI runs them, too.

Reply:
Sure, added. Using master might of course cause breakage from the pytest side, but hopefully not.